src/class-wpml-beaver-builder-register-strings.php

```diff
@@ -84,4 +84,13 @@ private function sort_modules_by_parent_and_child( array $all_modules, $parent_h
 	private function sort_modules_by_position_only( stdClass $a, stdClass $b ) {
 		return ( (int) $a->position < (int) $b->position ) ? -1 : 1;
 	}
-}
\ No newline at end of file
+
+	/**
+	 * @param object $data
```

Comment:
@strategio is `$data` an instance of `stdClass` or a specific object?

Reply:
I believe it's a `stdClass` @andreasciamanna, mostly because it's coming from unserialized data, but I am not 100% sure. I preferred not to use a type hint if this is what think about.

Reply:
Ok. I wasn't asking for the type-hint.

I was asking because I don't like the `! empty( .... )` below.

If a class has a property, it's always set, even if it falsy.

So I would use `$data->template_node_id && $data->template_node_id !== $data->node;` there.

But since it's an instance of `stdClass`, we can't be sure the object always has the property, so I guess we must live with that :)

Anyway, +1